2018-golab/MiningPatterns.md

```diff
@@ -0,0 +1,17 @@
+# Mining GoLang patterns
+
+**Updates:** https://github.com/src-d/conferences/issues/33
+
+**Sent on:** not sent
+
+**Status:** thinking
+
+**Author:** Vadim? Francesc?
+
+**Abstract:**
+
+I thought that since we've got our deduplication engine and many famous Go repositories, it will be possible to find the functions or snippets which are widely repeated throughout codebases. So we could talk about the results and how we made it.
+
+**Notes:**
+
+Personally, I am offended that the tracks are pretty much fixed and there is no room for Machine Learning - a super hot topic everybody is crazy about (apparently not in Italy).
```

Comment:
It's a Go conference 😄 
But yeah, I think it's an advantage for us, since this is topic that others will find different and interesting